services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/CopyObjectHelper.java

```diff
@@ -136,7 +135,7 @@ private void doCopyInParts(CopyObjectRequest copyObjectRequest,
                                                                                     uploadId,
                                                                                     completedParts,
                                                                                     optimalPartSize);
-        CompletableFutureUtils.allOfCancelForwarded(futures.toArray(new CompletableFuture[0]))
+        CompletableFutureUtils.allOfExceptionForwarded(futures.toArray(new CompletableFuture[0]))
```

Comment:
I find this gross to read, personally, but it's no worse than it already is, so I'm not going to be a jerk. I'll just make the passive-aggressive CR comment about it.

Reply:
Naively, more functions or variables so that indentation isn't so wide. Admittedly, I haven't put much thought.

Reply:
Oh, I thought it was the method name that you didn't like. I can clean it up a bit.

Reply:
https://github.com/aws/aws-sdk-java-v2/pull/3552